presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -58,12 +58,7 @@ private static void execute(String url, String sql)
 
     public String getJdbcUrl()
     {
-        return getJdbcUrl(this.dockerContainer::getHostPort);
-    }
-
-    private String getJdbcUrl(DockerContainer.HostPortProvider dockerContainer)
-    {
-        return format("jdbc:postgresql://localhost:%s/%s?user=%s&password=%s", dockerContainer.getHostPort(POSTGRESQL_PORT), DATABASE, USER, PASSWORD);
+        return format("jdbc:postgresql://%s:%s/%s?user=%s&password=%s", dockerContainer.getContainerIpAddress(), dockerContainer.getMappedPort(POSTGRESQL_PORT), DATABASE, USER, PASSWORD);
```

Comment:
In tests we should configure PostgreSQL connector the way we expect user to do it.
In particular, we expect users to set `connection-user` and `connection-password` rather than encoding them in the URL.

I am aware this is pre-existing, so let's add TODO for now.

Reply:
see also https://github.com/prestosql/presto/pull/2138#discussion_r352539553